tests/test_utils_other.py

```diff
@@ -541,6 +544,31 @@ def test_is_almost_int():
     assert is_almost_int(-1.001, .1)
 
 
+def test_maybe_zero():
+    assert maybe_zero(0.0001, 0.1) == 0
+    assert maybe_zero(-0.0001, 0.1) == 0
+    assert maybe_zero(1.5, 0.1) == 1.5
+
+
+def test_maybe_int():
+    assert maybe_int(1, 1e-10) == 1
```

Comment:
Needs tests that verify that "same object" is returned when there is no truncation

```python
for x in [3/7, 7/3, -13.7878]:
    assert maybe_int(x) is x
```

Reply:
I'll add this in